lib/gooddata/core/rest.rb

```diff
@@ -94,6 +101,13 @@ def upload_to_project_webdav(file, options = { :project => GoodData.project })
     def download_from_project_webdav(file, where, options = { :project => GoodData.project })
       options = merge_options(options)
       url = project_webdav_path(options)
+
+      unless url
+        msg = 'Project WebDAV not supported in this Data Center'
+        GoodData.logger.error msg
+        raise msg
+      end
+
```

Comment:
this code should probably go to project_webdav_path .. so that it's not duplicated here.
The logging should probably be done automatically when you raise an error like that - I've added a comment to BIA-27